core/client/app/templates/components/gh-profile-image.hbs

```diff
@@ -16,5 +16,5 @@
             <span class="sr-only">Upload an image</span>
         </i>
     </span>
-    <input type="file" class="file-uploader js-file-input" name="uploadimage">
+    {{#if fileStorage}}<input type="file" class="file-uploader js-file-input" name="uploadimage">{{/if}}
```

Comment:
I don't think prepending `has` onto it makes it better or worse.  Seems fine the way it is.

I will say that if we pass something into a component, we should declare it in the component itself.  It's nice to know what the public interface to a component is when you're working in the file, and in the future there's actually going to be a difference between things that are declared in a component, and things that are only passed in during invocation.  So in that regard it's probably safer to always declare things in the component to avoid weirdness down the road.